src/main/java/uk/gov/hmcts/reform/finrem/documentgenerator/service/DocumentConversionService.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.finrem.documentgenerator.service;
+
+
+
+
+import com.google.common.io.Files;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.finrem.documentgenerator.error.DocumentConversionException;
+import uk.gov.hmcts.reform.finrem.documentgenerator.model.Document;
+
+import java.io.File;
+import java.io.IOException;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class DocumentConversionService {
+
+    private static final String PDF = "pdf";
+
+    @Value("${service.pdf-service.uri}/rs/convert")
+    private String documentConversionUrl;
+
+    @Value("${service.pdf-service.accessKey}")
+    private String docmosisAccessKey;
+
+    private final RestTemplate restTemplate;
+
+    private final EvidenceManagementService evidenceManagementService;
+
+
+    public byte[] convertDocumentToPdf(Document sourceDocument) {
+        if (sourceDocument.getFileName().toLowerCase().endsWith(PDF)) {
+            throw new DocumentConversionException(
+                "Document already is a pdf",
+                null
+            );
+        }
+
+        return convert(sourceDocument, PDF);
+    }
+
+    public String getConvertedFilename(String filename) {
+        return filename.split("\\.")[0] + ".pdf";
```

Comment:
Better use something from `org.apache.commons.io.FilenameUtils` - the service has the library already included. This version would return eg. "order.pdf" for "order.contested.doc"